tests/cmd/builds/commands.go

```diff
@@ -19,16 +19,20 @@ const ExampleImage = "deis/example-dockerfile-http"
 
 // Create executes `deis builds:create` as the specified user.
 func Create(user model.User, app model.App) {
-	createOrPull(user, app, "builds:create")
+	createOrPull(user, app, "builds:create", "")
 }
 
 // Pull executes the `deis pull` shortcut as the specified user.
-func Pull(user model.User, app model.App) {
-	createOrPull(user, app, "pull")
+func Pull(user model.User, app model.App, image string) {
+	createOrPull(user, app, "pull", image)
 }
 
-func createOrPull(user model.User, app model.App, command string) {
-	sess, err := cmd.Start("deis %s --app=%s %s", &user, command, app.Name, ExampleImage)
+func createOrPull(user model.User, app model.App, command string, image string) {
```

Comment:
imo, this could be implemented more cleanly using variadic arguments.

Method signature:

```
func createOrPull(user model.User, app model.App, command string, opts ...string)
```
`Create(...)` and `Pull(...)` functions can have their signatures modified similarly.

By doing this, invocations of the `Create(...)` or `Pull(...)` functions will not require the awkward `""` at the tail end of the argument list.  Specifying an image would be fully optional.  `createOrPull(...)` can retrieve it from `opts[0]`.